### ISSUE TITLE
build app on ios problem

### ISSUE THREAD

**Issue by EngMo974** [inspireui/support#4994]
![تعليق توضيحي 2020-09-13 233637](https://user-images.githubusercontent.com/42417845/93028063-fdbf2800-f619-11ea-8f17-777ea013fadc.png)

**Comment by minhcasi** [comment]
@EngMo974 - could you try some clean cache solution on this link - https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1 - make sure to update the `pod repo update` on your project

**Comment by EngMo974** [comment]
hello, I tried this alot of times but still doesnt work https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1 
that is all of consol

━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
Warning
──────────────────────────────────────────────────────────────────────────────
Your Flutter application is created using an older version of the Android
embedding. It's being deprecated in favor of Android embedding v2. Follow the
steps at

https://flutter.dev/go/android-project-migration

to migrate your project.
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

Launching lib/main.dart on iPhone SE (2nd generation) in debug mode...
Running pod install...
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)
      Using `ARCHS` setting to build architectures of target `Pods-OneSignalNotificationServiceExtension`: (``)

    Finding Podfile changes
      - Flutter
      - OneSignal
      - app_settings
      - apple_sign_in
      - connectivity
      - facebook_audience_network
      - firebase_admob
      - firebase_analytics
      - firebase_auth
      - firebase_core
      - firebase_database
      - firebase_messaging
      - flutter_facebook_login
      - flutter_statusbarcolor
      - flutter_webview_plugin
      - google_maps_flutter
      - google_sign_in
      - location
      - notification_permissions
      - onesignal_flutter
      - path_provider
      - rate_my_app
      - screen
      - share
      - shared_preferences
      - sqflite
      - uni_links
      - url_launcher
      - video_player
      - wakelock
      - webview_flutter

    Fetching external sources
    -> Fetching podspec for `Flutter` from `Flutter`
    -> Fetching podspec for `app_settings` from `.symlinks/plugins/app_settings/ios`
    -> Fetching podspec for `apple_sign_in` from `.symlinks/plugins/apple_sign_in/ios`
    -> Fetching podspec for `connectivity` from `.symlinks/plugins/connectivity/ios`
    -> Fetching podspec for `facebook_audience_network` from `.symlinks/plugins/facebook_audience_network/ios`
    -> Fetching podspec for `firebase_admob` from `.symlinks/plugins/firebase_admob/ios`
    -> Fetching podspec for `firebase_analytics` from `.symlinks/plugins/firebase_analytics/ios`
    -> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
    -> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
    -> Fetching podspec for `firebase_database` from `.symlinks/plugins/firebase_database/ios`
    -> Fetching podspec for `firebase_messaging` from `.symlinks/plugins/firebase_messaging/ios`
    -> Fetching podspec for `flutter_facebook_login` from `.symlinks/plugins/flutter_facebook_login/ios`
    -> Fetching podspec for `flutter_statusbarcolor` from `.symlinks/plugins/flutter_statusbarcolor/ios`
    -> Fetching podspec for `flutter_webview_plugin` from `.symlinks/plugins/flutter_webview_plugin/ios`
    -> Fetching podspec for `google_maps_flutter` from `.symlinks/plugins/google_maps_flutter/ios`
    -> Fetching podspec for `google_sign_in` from `.symlinks/plugins/google_sign_in/ios`
    -> Fetching podspec for `location` from `.symlinks/plugins/location/ios`
    -> Fetching podspec for `notification_permissions` from `.symlinks/plugins/notification_permissions/ios`
    -> Fetching podspec for `onesignal_flutter` from `.symlinks/plugins/onesignal_flutter/ios`
    -> Fetching podspec for `path_provider` from `.symlinks/plugins/path_provider/ios`
    -> Fetching podspec for `rate_my_app` from `.symlinks/plugins/rate_my_app/ios`
    -> Fetching podspec for `screen` from `.symlinks/plugins/screen/ios`
    -> Fetching podspec for `share` from `.symlinks/plugins/share/ios`
    -> Fetching podspec for `shared_preferences` from `.symlinks/plugins/shared_preferences/ios`
    -> Fetching podspec for `sqflite` from `.symlinks/plugins/sqflite/ios`
    -> Fetching podspec for `uni_links` from `.symlinks/plugins/uni_links/ios`
    -> Fetching podspec for `url_launcher` from `.symlinks/plugins/url_launcher/ios`
    -> Fetching podspec for `video_player` from `.symlinks/plugins/video_player/ios`
    -> Fetching podspec for `wakelock` from `.symlinks/plugins/wakelock/ios`
    -> Fetching podspec for `webview_flutter` from `.symlinks/plugins/webview_flutter/ios`

    Resolving dependencies of `Podfile`
      CDN: trunk Relative path: CocoaPods-version.yml exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_c_1_2.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.15.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_c_7_9.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/7/9/Reachability/3.2/Reachability.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_0_3_5.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.32.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_9_b_5.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/7.1.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_b_3_c.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/7.1.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_d_4_0.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/d/4/0/GoogleSignIn/5.0.2/GoogleSignIn.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_a_d_d.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.10.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_2_1_5.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/1/5/FBAudienceNetwork/5.10.1/FBAudienceNetwork.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_f_4_e.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_5_9_a.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/5/9/a/Google-Mobile-Ads-SDK/7.65.0/Google-Mobile-Ads-SDK.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/5/9/a/Google-Mobile-Ads-SDK/7.63.0/Google-Mobile-Ads-SDK.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_e_3_b.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.8.1/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.7.0/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_0_8_4.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.2/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_6_1_e.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30906.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/1/e/nanopb/1.30905.0/nanopb.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_6_3_6.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/3/6/FirebaseAuth/6.9.1/FirebaseAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/6/3/6/FirebaseAuth/6.7.1/FirebaseAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_8_b_d.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.10.2/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_c_e_3.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/3/5/Firebase/6.29.0/Firebase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_e_2_1.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/2/1/FirebaseAnalytics/6.8.1/FirebaseAnalytics.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.14.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/1/2/OneSignal/2.14.3/OneSignal.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.9.2/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_8_3_c.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/8/3/c/FirebaseCoreDiagnostics/1.6.0/FirebaseCoreDiagnostics.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/8/b/d/FirebaseCore/6.9.2/FirebaseCore.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_0_b_5.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/b/5/PromisesObjC/1.2.10/PromisesObjC.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_2_d_6.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/d/6/FirebaseMessaging/4.6.2/FirebaseMessaging.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_d_8_2.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/d/8/2/FirebaseDatabase/6.6.0/FirebaseDatabase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/1/5/FBAudienceNetwork/5.8.0/FBAudienceNetwork.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/7/9/Reachability/3.2/Reachability.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/b/5/PromisesObjC/1.2.9/PromisesObjC.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/5.15.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/3/c/FBSDKLoginKit/5.15.1/FBSDKLoginKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/f/4/e/FMDB/2.7.5/FMDB.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/d/8/2/FirebaseDatabase/6.4.0/FirebaseDatabase.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_a_d_a.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/a/leveldb-library/1.22/leveldb-library.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/d/GoogleMaps/3.9.0/GoogleMaps.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/2/1/FirebaseAnalytics/6.7.0/FirebaseAnalytics.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_2_f_7.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.7.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/3/b/GoogleAppMeasurement/6.7.0/GoogleAppMeasurement.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.5.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/8/3/c/FirebaseCoreDiagnostics/1.5.0/FirebaseCoreDiagnostics.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_0_6_a.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/6/a/GoogleDataTransport/7.3.0/GoogleDataTransport.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/a/d/a/leveldb-library/1.22/leveldb-library.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/d/6/FirebaseMessaging/4.6.1/FirebaseMessaging.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_3_6_0.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/3/6/0/FirebaseInstanceID/4.6.0/FirebaseInstanceID.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/8/4/GoogleUtilities/6.7.1/GoogleUtilities.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_e_c_d.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/c/d/Protobuf/4.0.0-rc1/Protobuf.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/3/6/0/FirebaseInstanceID/4.5.1/FirebaseInstanceID.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/f/7/FirebaseInstallations/1.5.0/FirebaseInstallations.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/9/b/5/FBSDKCoreKit/5.15.1/FBSDKCoreKit.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/d/4/0/GoogleSignIn/5.0.2/GoogleSignIn.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_b_b_9.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: all_pods_versions_2_c_c.txt exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/c/c/GTMAppAuth/1.0.0/GTMAppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/0/6/a/GoogleDataTransport/7.1.0/GoogleDataTransport.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/e/c/d/Protobuf/3.12.0/Protobuf.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/2/c/c/GTMAppAuth/1.0.0/GTMAppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/b/b/9/AppAuth/1.4.0/AppAuth.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update
      CDN: trunk Relative path: Specs/c/e/3/GTMSessionFetcher/1.4.0/GTMSessionFetcher.podspec.json exists! Returning local because checking is only perfomed in repo update

    Comparing resolved specification to the sandbox manifest
      A AppAuth
      A FBAudienceNetwork
      A FBSDKCoreKit
      A FBSDKLoginKit
      A FMDB
      A Firebase
      A FirebaseAnalytics
      A FirebaseAuth
      A FirebaseCore
      A FirebaseCoreDiagnostics
      A FirebaseDatabase
      A FirebaseInstallations
      A FirebaseInstanceID
      A FirebaseMessaging
      A Flutter
      A GTMAppAuth
      A GTMSessionFetcher
      A Google-Mobile-Ads-SDK
      A GoogleAppMeasurement
      A GoogleDataTransport
      A GoogleMaps
      A GoogleSignIn
      A GoogleUtilities
      A OneSignal
      A PromisesObjC
      A Protobuf
      A Reachability
      A app_settings
      A apple_sign_in
      A connectivity
      A facebook_audience_network
      A firebase_admob
      A firebase_analytics
      A firebase_auth
      A firebase_core
      A firebase_database
      A firebase_messaging
      A flutter_facebook_login
      A flutter_statusbarcolor
      A flutter_webview_plugin
      A google_maps_flutter
      A google_sign_in
      A leveldb-library
      A location
      A nanopb
      A notification_permissions
      A onesignal_flutter
      A path_provider
      A rate_my_app
      A screen
      A share
      A shared_preferences
      A sqflite
      A uni_links
      A url_launcher
      A video_player
      A wakelock
      A webview_flutter

    Downloading dependencies

    -> Installing AppAuth (1.4.0)
      > Copying AppAuth from `/Users/ami/Library/Caches/CocoaPods/Pods/Release/AppAuth/1.4.0-31bce` to `Pods/AppAuth`

    -> Installing FBAudienceNetwork (5.8.0)
     > Http download
       $ /usr/bin/curl -f -L -o /var/folders/47/rck5ft7d1m7bbxyb2csfj8lr0000gn/T/d20200913-13843-5n0bts/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

    [!] Error installing FBAudienceNetwork
    [!] /usr/bin/curl -f -L -o /var/folders/47/rck5ft7d1m7bbxyb2csfj8lr0000gn/T/d20200913-13843-5n0bts/file.zip https://developers.facebook.com/resources/FBAudienceNetwork-5.8.0.zip --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.9.3 cocoapods-downloader/1.4.0'

      % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                     Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
    Warning: Transient problem: HTTP error Will retry in 1 seconds. 2 retries 
    Warning: left.
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
    Warning: Transient problem: HTTP error Will retry in 2 seconds. 1 retries 
    Warning: left.
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
    curl: (22) The requested URL returned error: 500 

    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:142:in `rescue in execute_command'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:139:in `execute_command'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:166:in `block in executable'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/http.rb:22:in `download_file'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/remote_file.rb:25:in `download!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:82:in `block in download'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:157:in `block in ui_action'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:156:in `ui_action'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-downloader-1.4.0/lib/cocoapods-downloader/base.rb:80:in `download'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:108:in `download_source'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:75:in `download_request'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:171:in `download'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:154:in `block in uncached_pod'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:181:in `in_tmpdir'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:153:in `uncached_pod'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader/cache.rb:33:in `download_pod'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/downloader.rb:42:in `download'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer/pod_source_installer.rb:117:in `download_source'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer/pod_source_installer.rb:69:in `install!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:557:in `install_source_of_pod'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:508:in `block (2 levels) in install_pod_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:86:in `titled_section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:507:in `block in install_pod_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:490:in `each'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:490:in `install_pod_sources'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:249:in `block in download_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:248:in `download_dependencies'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/installer.rb:157:in `install!'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/command/install.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.6.0/gems/cocoapods-1.9.3/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:23:in `load'
    /usr/local/bin/pod:23:in `<main>'

Error output from CocoaPods:
↳
         % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                        Dload  Upload   Total   Spent    Left  Speed
     0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
       Warning: Transient problem: HTTP error Will retry in 1 seconds. 2 retries 
       Warning: left.
     0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
       Warning: Transient problem: HTTP error Will retry in 2 seconds. 1 retries 
       Warning: left.
     0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
       curl: (22) The requested URL returned error: 500 

    [!] Automatically assigning platform `iOS` with version `10.3` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

    [!] Automatically assigning platform `iOS` with version `10.3` on target `OneSignalNotificationServiceExtension` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error running pod install
Error launching application on iPhone SE (2nd generation).
◦

**Comment by EngMo974** [comment]
Still I need help, quickly

![تعليق توضيحي 2020-09-14 152437](https://user-images.githubusercontent.com/42417845/93085714-a9aa5700-f69e-11ea-8817-a2a8f17f436d.png)

No valid code signing certificates were found
You can connect to your Apple Developer account by signing in with your Apple ID
in Xcode and create an iOS Development Certificate as well as a Provisioning 
Profile for your project by:
  1- Open the Flutter project's Xcode target with
       open ios/Runner.xcworkspace
  2- Select the 'Runner' project in the navigator then the 'Runner' target
     in the project settings
  3- Make sure a 'Development Team' is selected. 
     - For Xcode 10, look under General > Signing > Team.
     - For Xcode 11 and newer, look under Signing & Capabilities > Team.
     You may need to:
         - Log in with your Apple ID in Xcode first
         - Ensure you have a valid unique Bundle ID
         - Register your device with your Apple Developer Account
         - Let Xcode automatically provision a profile for your app
  4- Build or run your project again
  5- Trust your newly created Development Certificate on your iOS device
     via Settings > General > Device Management > [your new certificate] > Trust

**Comment by qozia3** [comment]
use a vpn to solve the first problem. 
it will work

**Comment by qozia3** [comment]
use a vpn to solve the first problem before starting the app or running pod install
it will work

**Comment by minhcasi** [comment]
@EngMo974 - try to check this option - it will help to auto register your profile with apple
<img width="782" alt="Screen Shot 2020-09-15 at 13 54 06" src="https://user-images.githubusercontent.com/1459805/93176165-09fece80-f75b-11ea-995f-161373a60598.png">

Also, use VPN if you could download the library from Pod package.

**Comment by EngMo974** [comment]
I tried this, and I don't need any sign in methods, like apple, google, facebook and sms, so I need help to that focus on apple method, It makes millions of problems, I need help here, please pay attention

**Comment by minhcasi** [comment]
@EngMo974 - try to clean up cache and upgrade Pod file library to fix the issue - please also refer to - https://support.inspireui.com/help-center/articles/3/8/69/common-flutter-issues

**Comment by EngMo974** [comment]
@minhcasi why u dont understand me, idont need any src code in my src relates with apple sign in, that is not hard.

**Comment by minhcasi** [comment]
Could you able to build the Xcode and run on the iphone simulator so far?

**Comment by EngMo974** [comment]
@minhcasi  no I cant build or run on simulator even physical device

**Comment by minhcasi** [comment]
@EngMo974 - pls try a try to run the Test Drive app to see if you could run it smoothly on XCode? https://flutter.dev/docs/get-started/test-drive

**Comment by minhcasi** [comment]
Please move the new conversation to new Ticket Support http://support.inspireui.com - thank you!
